stdlib/public/runtime/ReflectionMirror.cpp

```diff
@@ -1092,3 +1092,28 @@ id swift_reflectionMirror_quickLookObject(OpaqueValue *value, const Metadata *T)
 #endif
 
 #endif  // SWIFT_ENABLE_REFLECTION
+
+#include "ImageInspection.h"
+
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_API
```

Comment:
These should probably be `SWIFT_RUNTIME_STDLIB_INTERNAL`.

Reply:
Thanks, I didn't know that was an option. I will try switching.